@here/olp-sdk-dataservice-read/lib/DownloadManager.ts

```diff
@@ -18,22 +18,20 @@
  */
 
 /**
- * DownloadManager for downloading URLs.
- *
- * Interface for DataStoreRequestBuilder.
- *
- * DataStoreRequestBuilder has own default implementation of download manager.
- *
- * If you want to use your own download manager, you are able implement this interface end set it.
+ * Downloads URLs.
+ * An interface for `DataStoreRequestBuilder`.
+ * `DataStoreRequestBuilder` has its own default implementation of the download manager.
+ * If you want to use your download manager, you can implement this interface, and then set your download manager.
```

Comment:
it's not for Downloads URLs and not an interface for `DataStoreRequestBuilder`. It's an interface for @here/olp-sdk-dataservice-api to be able to send requests to the services

Reply:
Fixed